### DISCUSSION TITLE
Snaps in token approve transaction

### DISCUSSION THREAD

**Discussion by highoak** [MetaMask/snaps#904] [Q&A] [answered]
I noticed that in token approve transaction there is no option to run snaps like any other transaction. Is it a bug or a feature?
Is there any way to run it also in approve?
![image](https://user-images.githubusercontent.com/9609659/199720211-5e386942-1acd-4eff-b3bb-93379a5d97f6.png)

**Comment by ritave** [comment, marked as answer]
There's no way to put a tab there, and so for now it's not supported.

I've created an issue (#907) and will try to get this functionality in in the future.
